docs/source/basics/tokenizers.rst

```diff
@@ -0,0 +1,320 @@
+.. _tokenizers_usage_label:
+
+==========
+Tokenizers
+==========
+
+Tokenizers are a key component of any LLM. They convert raw text into token IDs, which index into embedding vectors that are
+understood by the model.
+
+In torchtune, tokenizers play the role of converting Messages object into token IDs and any necessary model-specific special tokens.
+They use byte-pair encoding algorithms to convert raw strings into integer IDs. These are usually based on SentencePiece or TikToken.
+
+Downloading tokenizers from Hugging Face
+----------------------------------------
+
+Use the ``tune download`` command to download tokenizers from Hugging Face. They are found on the model page and are downloaded with
```

Comment:
maybe
"Models hosted on Hugging Face are also distributed with the tokenizers they were trained with. These are automatically downloaded alongside model weights when using ``tune download``. For exmple, this command downloads the Mistral-7B model weights and tokenizer:"


and maybe also? But maybe this isn't a common usecase
If you're just interested in download the tokenizer without any model weights, you can use --ignore-patterns "..."